deploy/genesis/docker-compose-ganache.yml

```diff
@@ -9,25 +9,19 @@ networks:
 services:
```

Comment:
We should also move this file under `./test/**/*` somewhere/ too

Reply:
Also moved to test/integration